source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -14,163 +15,159 @@
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
-
    * - ext 1.12 + lib 1.11
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-     -
-     -
-     -
-
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
+     - 
+     - 
+     - 
    * - ext 1.11 + lib 1.10
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - 
    * - ext 1.10 + lib 1.9
-     -
-     - |checkmark| [#PHPC1.10-PHPLIB1.9-driver-support]_
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     -
-
+     - 
+     - 
+     - ✓ [#PHPC1.10-PHPLIB1.9-driver-support]_
```

Comment:
This isn't [rendering correctly.](https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-20488-mongo-52-compat/php/#mongodb-compatibility)

Reply:
Sorry I missed that, thanks for catching.